src/iris/api.py

```diff
@@ -3257,7 +3261,7 @@ def get_api(config):
     master_sender_addr = (master_sender['host'], master_sender['port'])
     # all notifications go through master sender for now
     app = construct_falcon_api(
```

Comment:
https://github.com/linkedin/iris/blob/master/src/iris/doc_helper.py needs to be updated too otherwise the document build will fail.